lib/google_scraping_web/controllers/api/v1/auth_controller.ex

```diff
@@ -0,0 +1,24 @@
+defmodule GoogleScrapingWeb.Api.V1.AuthController do
+  use GoogleScrapingWeb, :controller
+
+  alias GoogleScraping.Account.Guardian
+  alias GoogleScraping.Accounts
+  alias GoogleScrapingWeb.Api.V1.ErrorView
+
+  def create(conn, %{"email" => email, "password" => password}) do
```

Comment:
Since this is an API, invalid params could be more likely. 
If we put an invalid params, it would raise a pattern match error right? I think that would return a 500 error by default 💭 

Should we have a pattern match after this one to return some expected error like 422?

Reply:
Thanks for the good suggestion. It was fixed here: https://github.com/Nihisil/nimble-certification-elixir/pull/54/commits/0ec74682d7e7ddae4ea10d4ecb70d574822e2528